src/rgw/setup/rgw.py

```diff
@@ -494,48 +494,46 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
         # 1. While creating admin user, global lock created in consul kv store.
         # (rgw_consul_index, cortx>rgw>volatile>rgw_lock, machine_id)
         # 2. Before creating admin user.
-        #    a. Check for rgw_lock in consul kv store.
-        #    b. Create user only if lock value is None/machine-id.
+        #    a. Check for rgw_lock in consul kv store
+        #    b. Create user only if lock value is equal to machine_id
```

Comment:
#    b. Create user only if lock value is equal to **self** machine_id
        # 3. If user creation attempt failed from this node, delete the lock so other node can acquire the lock
        #    and try user creation.